autoload/go/lsp/message.vim

```diff
@@ -325,7 +325,11 @@ function! go#lsp#message#ConfigurationResult(items) abort
     endif
 
     if l:gofumpt isnot v:null
-      let l:config.gofumpt = l:gofumpt
+      if l:config.gofumpt
```

Comment:
Shouldn't this check `l:gofumpt`?

Getting this error at the moment when setting `g:go_gopls_gofumpt`: `E716: Key not present in Dictionary: gofumpt`

Reply:
🤦 PR is to fix it is up now: #3022